### PR TITLE
Drop default hash value

### PR DESCRIPTION
**Pull request by DukeManh**
## Issue This PR Addresses
Drop the default value for Feeds hash ids now that all feeds have an id

## Type of Change

- [ ] **Bugfix**: Change which fixes an issue
- [ ] **New Feature**: Change which adds functionality
- [ ] **Documentation Update**: Change which improves documentation
- [ ] **UI**: Change which improves UI

## Checklist

- [ ] **Quality**: This PR builds and passes our npm test and works locally
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [ ] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

File: src/db/prisma/schema.prisma [hunk 1]
```diff
@@ -16,14 +16,16 @@ enum FeedType {
 // Planet CDOT Feed List, see https://wiki.cdot.senecacollege.ca/wiki/Planet_CDOT_Feed_List
 model feeds {
   url                String              @id
-  id                 String              @default("")
+  id                 String  // A short hashed id generated from the URL be Satellite,
```
Comment (TueeNguyen) [inline src/db/prisma/schema.prisma:19]: I don't understand this `from the URL be Satellite`

**Review by TueeNguyen — DISMISSED** [review]

**Review by tcvan0707 — APPROVED** [review]

**Comment by TueeNguyen** [comment]
I guess the some sso tests fails because of the new `hash` change?

**Review by JerryHue — APPROVED** [review]

File: src/api/sso/test/e2e/supabase-util.js [hunk 2]
```diff
@@ -70,6 +71,7 @@ const addFeeds = (feeds) =>
     .insert(
       feeds.map((feed) => ({
         url: feed.url,
+        id: hash(normalizeUrl(feed)),
```
Comment (JerryHue) [inline src/api/sso/test/e2e/supabase-util.js:74]: As per CI:
```suggestion
        id: hash(normalizeUrl(feed.url)),
```

**Review by JerryHue — CHANGES_REQUESTED** [review]

**Comment by humphd** [comment]
> I guess the some sso tests fail because of the new `hash` change?

Ah, so this is why we're breaking!

**Comment by DukeManh** [comment]
I was away to push the new commit, the tests should pass now.

**Comment by DukeManh** [comment]
@humphd, the checks from you pr should be a different thing because the `id` is optional for now.

**Comment by humphd** [comment]
Except you're failing the same test as I am:

```
 ✕ signup flow works to login and register a new Telescope user (20215 ms)
    ✓ signup flow fails if user is not authenticated (130 ms)
    ✓ signup flow fails if user data is missing required properties (807 ms)
    ✓ signup flow fails if user is already a Telescope user (672 ms)
    ✓ Signup flow fails if feed is already used by another Telescope user (1346 ms)

  ● Signup Flow › signup flow works to login and register a new Telescope user

    thrown: "Exceeded timeout of 20000 ms for a test.
    Use jest.setTimeout(newTimeout) to increase the timeout value, if this is a long-running test."

      64 |     ensureUsers(users, false));
      65 |
    > 66 |   it('signup flow works to login and register a new Telescope user', async () => {
         |   ^
      67 |     // Part 1: login using SSO, as a user who does not have a Telescope account.
      68 |     // Confirm that the payload of the token we get back matches what we expect.
      69 |     // NOTE: the data comes from config/simplesamlphp-users.php.

      at src/api/sso/test/e2e/signup-flow.test.js:66:3
      at Object.<anonymous> (src/api/sso/test/e2e/signup-flow.test.js:46:1)
```

**Comment by DukeManh** [comment]
Now we share the error. What's going on.

**Comment by humphd** [comment]
I blame @Kevan-Y

**Review by humphd — APPROVED** [review]

**Review by TueeNguyen — APPROVED** [review]

**Pull request merged**